[22308] Ensure Search for Inclusions works if script is unlocked

livecodepanos opened the pull request.

livecodeali commented on ide-support/revsblibrary.livecodescript:
```diff
@@ -2875,6 +2875,15 @@ private command revSmartChecking pStackFilesList, @xSettings
+      return true
+   end try
+end utilityScriptIsLocked
+
```
For other private utilities in this file we use the convention of beginning the handler name with double underscore, so I suggest just calling this `__ScriptIsLocked`.
I would also put the `return false` after the try I think

livecodeali commented:
@livecodepanos Also could you add a commit message please?

livecodepanos committed:
[22308] Ensure Search for Inclusions works if script is unlocked

This patch checks if the script if locked before searching for inclusions,
rather than checking if the stack has a password. If the script is locked,
then the auto-search cannot be done and a warning is thrown.

livecodeali commented:
@livecode-vulcan review ok c1ca132

livecode-vulcan commented:
💙 review by @livecodeali ok c1ca132f6760fa2152c08a1c73ae1bf6cd2e728a

livecode-vulcan commented:
😎 test success c1ca132f6760fa2152c08a1c73ae1bf6cd2e728a

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/631)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/40)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/40)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/40)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/40)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2157)
* try-community-universal-ios-iphoneos12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos12.1/builds/382)
* try-community-universal-ios-iphonesimulator12.1: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator12.1/builds/378)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2186)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2173)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2172)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2206)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2146)
